pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -314,4 +315,17 @@ public void getPendingAckInternalStats(@Suspended final AsyncResponse asyncRespo
             resumeAsyncResponseExceptionally(asyncResponse, ex);
         }
     }
+
+    @POST
+    @Path("/transactionCoordinator/partitions")
+    @ApiResponses(value = {
+            @ApiResponse(code = 503, message = "This Broker is not configured "
+                    + "with transactionCoordinatorEnabled=true."),
+            @ApiResponse(code = 406, message = "The number of partitions should be more than "
+                    + "the current number of transaction coordinator partitions"
+                    + " and less than or equal to maxNumPartitionsPerPartitionedTopic")})
+    public void updateTransactionCoordinatorNumber(int numPartitions) {
+        checkTransactionCoordinatorEnabled();
```

Comment:
should add auth

Reply:
+1, only the super user can make this change.

Reply:
I have add the auth just like `updatePartitionedTopic` at last commit.  Are there other problems?